pkg/model/request_entity_test.go

```diff
@@ -79,54 +79,54 @@ func testRequestEntity(t *testing.T, describe spec.G, it spec.S) {
 		})
 	})
 
-	describe("NextBackoff()", func() {
-		var backoff time.Duration
-		var outOfAttempts bool
-
-		describe("scheduling the first retry", func() {
-			it.Before(func() {
-				subject = NewRequestEntity(envFake, bufferStock)
-				backoff, outOfAttempts = subject.NextBackoff()
-			})
-
-			it("returns the next backoff duration", func() {
-				assert.Equal(t, 100*time.Millisecond, backoff)
-			})
-
-			// yes, this test can pass by coincidence, but at least it documents the intent
-			it("returns false to indicate that more retries are possible", func() {
-				assert.False(t, outOfAttempts)
-			})
-		})
-
-		describe("scheduling subsequent retries", func() {
-			it.Before(func() {
-				subject = NewRequestEntity(envFake, bufferStock)
-				backoff, _ = subject.NextBackoff()
-			})
-
-			// The real implementation has jitter, but I will ignore this for now.
-			it("on each retry, increases the backoff time by 1.3x", func() {
-				backoff, _ = subject.NextBackoff()
-				assert.Equal(t, 130*time.Millisecond, backoff)
-				backoff, _ = subject.NextBackoff()
-				assert.Equal(t, 169*time.Millisecond, backoff)
-			})
-		})
-
-		describe("running out of retries", func() {
-			it.Before(func() {
-				subject = NewRequestEntity(envFake, bufferStock)
-				for i := 0; i < 18; i++ {
-					_, outOfAttempts = subject.NextBackoff()
-				}
-				assert.False(t, outOfAttempts)
-			})
-
-			it("returns true to indicate no more retries are possible", func() {
-				_, outOfAttempts = subject.NextBackoff()
-				assert.True(t, outOfAttempts)
-			})
-		})
-	})
+	//describe("NextBackoff()", func() {
```

Comment:
If these tests don't apply anymore, just delete them.

Reply:
Deleted them in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit.